### PR TITLE
Footer Enhancements

### PR DESCRIPTION
**Pull request by RobbieZhuang**
As per Slack requests:
- Update link hover colour (pink on purple + purple on pink)
- Push Footer to stay at the bottom of the page
- Add Footer to the Events page
- Content/Body links turn pink when hovering

Pink/Purple text are links that the mouse is currently hovering over:

**Events**
events
**About**
Screen Shot 2020-05-19 at 12 00 49 PM

**Note: When reviewing, you can hide whitespace changes.**

### CODE REVIEW

File: src/containers/About.jsx [hunk 1]
```diff
@@ -11,93 +11,95 @@ const profileImage = name =>
...
-            dinners and social events. We introduce you to upper-years and
-            lower-years by matching you with someone to chat over coffee. We
-            highlight the cool stuff that everyone in SE is up to by producing
-            our podcast, Sessions by SE.
-          </p>
-          <Container className="mt-4 pl-0">
-            <h1>Meet The Team</h1>
+      <div className="footer-to-bottom">
+        <body>
+          <Container className="mt-5 mb-5">
```
Comment (RobbieZhuang) [inline src/containers/About.jsx:16]: Hide whitespace changes when reviewing

File: src/components/styles/Footer.scss [hunk 2]
```diff
@@ -6,4 +7,19 @@
 
 .link {
     color: white;
+}
+
+.theme {
```
Comment (alexgirard) [inline src/components/styles/Footer.scss:12]: looks great!! can this also be applied to the links on the home page?

File: src/containers/Events.jsx [hunk 3]
```diff
@@ -22,30 +23,35 @@ class Events extends Component {
...
+                    target="_blank"
+                  >
+                    + Google Calendar
+                  </Button>
+                </div>
+              </Col>
+            </Row>
+          </Container>
+        </body>
+        <Footer color={"purple"} />
```
Comment (alexgirard) [inline src/containers/Events.jsx:53]: 😄

File: src/components/styles/Footer.scss (hunk 2 above)
Comment (RobbieZhuang) [reply src/components/styles/Footer.scss:12]: Yup, good call.

File: src/App.scss [hunk 4]
```diff
@@ -89,4 +89,24 @@ p {
...
+}
+
+
+/*
+    `footer-to-bottom` wraps the body and footer for each page.
+    It spaces the two containers out so that the footer is pushed 
+    to the bottom of the view if the body contents do not fill up 
+    the whole view. Otherwise, the footer will rest below the 
+    body content.
+*/
```
Comment (jenniferwills) [inline src/App.scss:101]: :D nice description

**Comment by jenniferwills** [comment]
Can you change the PR title/description to say Footer, you're not changing the navbar here

File: src/App.scss (hunk 4 above)
Comment (RobbieZhuang) [reply src/App.scss:101]: I learned from a frand that comments are good :-)

**Comment by RobbieZhuang** [comment]
Oops my bad

**Pull request merged**